docs/src/components/landing-page/TwitterShowcaseCards.vue

```diff
@@ -148,10 +156,15 @@ export default defineComponent({
       }
     }
 
+    // calculate how many tweets to show per carousel depending on the screen size
+    // e.g. if screen size is sm, show 2 tweets per carousel. Hence, grid template-columns: repeat(2, 1fr);
+    const carouselGridTemplateColumns = computed(() => ({ gridTemplateColumns: `repeat(${NUMBER_OF_TWEETS_PER_CAROUSEL[ Screen.name ]}, 1fr)` }))
```

Comment:
Ok, didn't mean this when I said CSS v-bind, but this is fine too
I was talking of this new cool feature: https://v3.vuejs.org/api/sfc-style.html#state-driven-dynamic-css